test/api/test-set.c

```diff
@@ -470,6 +470,36 @@ test_set_empty (void)
   hb_set_destroy (b);
 }
 
+static void
+test_set_delrange (void)
```

Comment:
Can you add coverage of a few more interesting scenarios:
- The case where [a, b] covers exactly one page and also where [a, b] covers exactly two or more pages.
- Del range where there is no page at a or b.

Reply:
Updated impl & added more test cases.